README.md

```diff
@@ -1,7 +1,82 @@
-## Notes
+# Zedd-App
 
-- Use CTRL+LMB on the Calendar to add a new block or split an existing one.
-- TimeSlice RMB -> "Eat Next/Prev" joins this slice with the next/prev, filling the gap.
-- If accessing JIRA requires a root certificate, add it to the windows root certificate store and it will be used.
+ The Zedd-app is a Grindstone-like application that automates booking in Clarity and simplifies working with multiple accounts. It operates as an interactive calendar, allowing you to schedule tasks and manage your work time efficiently.
+
+The goal is to make work planning more organized, stress-free and user-friendly. The app is built with Electron, TypeScript, React, MobX, and MUI.
+
+## Feature 📝
+
+- Task selection or input windows
+- Main window with movable and stretchable task tiles
+- Arrow buttons and buttons for changing days, weeks, or months
+- Reservation button
+- Trash button for deleting all reservations
+- Three dashes button in the upper right corner for accessing and customizing the application settings
+- Work time clock for the application with stop option
+- Name change button
+- Import button for importing data from the Clarity system.
+- Summary window for work hours in the bottom part of the interface
+- Submit button for sending the set information to the Clarity system with the option of automatic submission.
+- Tasks can be linked to Jira tasks
+
```

Comment:
Beschränke dich hier auf die wichtigsten Feature der App. Was macht diese App genau aus.
Z.B. `Tasks can be linked to Jira tasks` ist ein cooles Feature, wobei `Name change button` jetzt sehr detailiert ist und man gar nicht versteht, wozu man das braucht.
Andererseits ist `Task selection or input windows` zu allgemein, denn das könnte irgendeine App sein.

Fokusiere dich auf 10 Features, die wirklich die App ausmachen. 
Du kannst dich auch am Changelog orientieren: 
https://github.com/tobka777/zedd2/blob/main/CHANGELOG.md

Ich verstehe nicht, was du mit reservations meinst...

Hier sind die Punkte, die ich als Feature empfinde:
- Tasks exported to the Clarity system with the option of automatic submission.
- Export kann auch im Hintergrund erfolgen
- Work clock time
- Tasks can be linked to Jira tasks
- importing data from the Clarity system
- Summary window for work hours
- Arbeitszeit, Überstunden überblick
- Automatische Pausieren der Arbeitszeit, wenn Rechner inaktiv
- Forecast mit Ersatz Button möglich
- individuelle Namen für Task, die zum einem Clarity Task mit Kommentar zugeordnet werden können
- automatische Erkennung von Feiertagen nach Land und Bundesland